tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,5 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result is False
+    assert len(fatimah.inventory) == 3
```

Comment:
For completeness, we could check to ensure that all three items remain where they were (confirming no partial operations occurred).